src/test/java/elizCurtnazar/tests/tema8/pageObject/JavaScriptRedirectsPage.java

```diff
@@ -0,0 +1,54 @@
+package elizCurtnazar.tests.tema8.pageObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class JavaScriptRedirectsPage {
+    ChromeDriver driver;
+    WebDriverWait wait;
+    Wait<WebDriver> fluentWait;
+
+    public JavaScriptRedirectsPage(ChromeDriver driver) {
+        this.driver = driver;
+        //using WebDriverWait
+        wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+        //using FluentWait
+        fluentWait = new FluentWait<WebDriver>(driver)
+                .withTimeout(Duration.ofSeconds(1))
+                .pollingEvery(Duration.ofSeconds(8));
```

Comment:
It doesn't make any sense to try after 8 seconds when maximum timeout is 1 second. Please use a smaller unit than seconds for pollingEvery part as you used in other page classes. Thanks!